pkg/loadbalancer/loadbalancer.go

```diff
@@ -240,46 +240,74 @@ const (
 )
 
 // BackendStateFlags is the datapath representation of the backend flags that
-// are used in (lb{4,6}_backend.flags) to store backend state.
+// are used in (lb{4,6}_backend.flags) to store backend state (e.g. Active,
```

Comment:
Why does the `preferred` attribute need to be persisted to BPF maps if these backends are still hidden in the datapath?

Reply:
Semi-related to https://github.com/cilium/cilium/pull/19521#issuecomment-1112951830. The endpoint is preferred, but state is updated manually via CLI (e.g. from active to quarantine), in this case, there is no change in service level.

While the agent can perform reconciliation again after restart, such reconciliation could take some time, and I assume having details in BPF maps will help to avoid any potential disruption.